app/src/androidTest/java/br/com/catbag/gifreduxsample/ui/giflist/GifListActivityTest.java

```diff
@@ -94,35 +99,16 @@ public void pauseGifTest() {
         onView(withId(R.id.gif_image))
                 .perform(click());
 
+        View image = mActivityTestRule.getActivity().findViewById(R.id.gif_image);
+        Log.i("pauseGifTest", "before "+image.getWidth()+" W"+ image.getHeight()+ " H");
+        sleep(2000);
```

Comment:
To remeber, change all sleeps to idling resource....probably a issue to be opened
```
// Make sure Espresso does not time out
IdlingPolicies.setMasterPolicyTimeout(waitingTime * 2, TimeUnit.MILLISECONDS);
IdlingPolicies.setIdlingResourceTimeout(waitingTime * 2, TimeUnit.MILLISECONDS);

// Now we wait
IdlingResource idlingResource = new ElapsedTimeIdlingResource(waitingTime);
Espresso.registerIdlingResources(idlingResource);

// Stop and verify
onView(withId(R.id.toggle_button))
    .check(matches(withText(R.string.stop)))
    .perform(click());
onView(withId(R.id.result))
    .check(matches(withText(success ? R.string.success: R.string.failure)));

// Clean up
Espresso.unregisterIdlingResources(idlingResource);
```